etc/notebooks/bokeh_demo/bokeh_demo.ipynb

```diff
@@ -0,0 +1,222 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "urth": {
+     "dashboard": {
+      "layout": {
+       "col": 0,
+       "height": 5,
+       "row": 0,
+       "width": 12
+      }
+     }
+    }
+   },
+   "source": [
+    "Bokeh Dashboard Demo\n",
+    "---------------------\n",
+    "\n",
+    "This demo shows off an interactive visualization using [Bokeh](http://bokeh.pydata.org) for plotting, and Ipython interactors for widgets. It is nearly identical to the [demo notebook](https://github.com/bokeh/bokeh/blob/master/examples/plotting/notebook/interact_basic.ipynb) found in Bokeh's own repository."
```

Comment:
Also, call out that a user can switch to dashboard layout and view modes to interact with these widgets.